examples/browser-webpack/package.json

```diff
@@ -11,13 +11,17 @@
     "babel-core": "^6.24.1",
     "babel-loader": "^7.0.0",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-0": "^6.24.1",
```

Comment:
Wha does the `babel-preset-stage-0` do and why do we need it here?

Reply:
I'm not sure why it was included in the `.babelrc` https://github.com/ipfs/js-ipfs/blob/master/examples/browser-webpack/.babelrc